adagrams/game.py

```diff
@@ -1,13 +1,80 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2,
+    'I': 9, 'J': 1, 'K': 1, 'L': 4, 'M': 2, 'N': 6, 'O': 8, 'P': 2,
+    'Q': 1, 'R': 6, 'S': 4, 'T': 6, 'U': 4, 'V': 2, 'W': 2, 'X': 1,
+    'Y': 2, 'Z': 1
+}
+
 def draw_letters():
-    pass
+
+    # A temporary pool built from LETTER_POOL counts
+    pool = []
+    for letter, quantity in LETTER_POOL.items():
+        pool += [letter] * quantity
+    
+    hand = []
+    for _ in range(10):
+        idx = randint(0, len(pool) - 1)     # pick a random index
```

Comment:
Your code would be more self-documenting if your variable name described what you wrote in the comment "# pick a random index". Having a more descriptive name would allow you to remove the comment too.

```suggestion
        random_idx = randint(0, len(pool) - 1) 
```